rclcpp/src/rclcpp/node_interfaces/node_parameters.cpp

```diff
@@ -185,6 +188,102 @@ __lockless_has_parameter(
   return parameters.find(name) != parameters.end();
 }
 
+// see https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon
+#define ULP 100
```

Comment:
Is this define better than just putting it as the default argument in the declaration? I don't see this used but in one place...

Reply:
Done in https://github.com/ros2/rclcpp/pull/735/commits/d91b2c4a17e961a3d7b9acc963f4db9404b2cf60